docs/src/pages/reference/configuration/output.md

```diff
@@ -300,6 +300,13 @@ Default Value: `1000`.
 
 Use to specify the delay time for the mock. It can either be a fixed number or a function that returns a number.
 
+#### delayCalcRuntime
+
+Type: `boolean`.
+
+Gives you the possibility to have functions that are passed to `delay` to be
+executed at runtime rather than at build time.
```

Comment:
To be more precise, is it "generated timing" rather than build time?

```suggestion
executed at runtime rather than at code generated timing.
```

Reply:
Or would `executed at runtime rather than when the mocks are generated.` be better?

Reply:
I like the second one `executed at runtime rather than when the mocks are generated.`

Reply:
i am not sure about this name `delayCalcRuntime` for a name shouldn't it be `delayMockFunction` or something like that?

Reply:
I've updated the comment and the variable name.

Reply:
Even before the change, I was able to specify a function, but the function would be executed when it was automatically generated, so I would like to have the delay function executed when a mock is used, right?
`Runtime` is certainly misleading, so it is inappropriate to use it in the sense of "when a function is executed." Since the execution is delayed relative to the existing automatic generation, what about `delayFunctionLazyExecute`? I think it will become more clear.

Or how about changing it so that it doesn't run at the timing of automatic generation, and instead runs the delay function when mock is used in all cases? I think this idea is good because it eliminates the complexity of adding options and does not affect existing users.

Reply:
I'm happy to change this to remove the extra option and always call the function at runtime. However, this would be a breaking change as the previous implementation allows for functions which reference external variables etc. whereas inserting the function into the mock requires it to be self contained.

Note: I chose the name runtime as this is how the docs refer to [other functions](https://orval.dev/reference/configuration/output#operations) that delay the execution of the function e.g. "Which can take a function or the value directly. The function will be executed at runtime."

Reply:
Ok leave both functions.  It would be a breaking change.

Reply:
I hadn't thought of breaking changes yet. thank you for let me know. What about the name?

> what about delayFunctionLazyExecute? I think it will become more clear.

Reply:
Happy to change it to that, or even `delayFunctionLazy` might be descriptive enough (I'm happy either way)?

Reply:
Hmm. I prefer `delayFunctionLazyExecute`.

Reply:
> Hmm. I prefer `delayFunctionLazyExecute`.

Done!

Reply:
thank you for your great work 👍